spk/gentoo-chroot/Makefile

```diff
@@ -0,0 +1,93 @@
+SPK_NAME = gentoo-chroot
+SPK_VERS = 13.0
+SPK_REV = 1
+SPK_ICON = src/gentoo.png
+DSM_UI_DIR = app
+
+DEPENDS = 
+SPK_DEPENDS = "python>2.7.3-3"
+
+MAINTAINER = "MasterRoot24"
+DESCRIPTION = "Gentoo is a free operating system based on either Linux or FreeBSD that can be automatically optimized and customized for just about any application or need. Gentoo Chroot allows you to install the Gentoo OS inside your DiskStation, alongside DSM. This package is intended for advanced users only."
+RELOAD_UI = yes
+DISPLAY_NAME = "Gentoo Chroot"
+CHANGELOG = 
+
+HOMEPAGE   = "http://www.gentoo.org/"
+LICENSE    = 
+HELPURL    = 
+HELPURL    = "3rdparty/gentoo-chroot/help/enu/index.html"
+SUPPORTURL = "http://www.gentoo.org/main/en/support.xml"
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+COPY_TARGET = nop
+POST_STRIP_TARGET = gentoo-chroot_extra_install
+
+GENTOO_ARCH =
+ifeq ($(findstring $(ARCH),88f6281),$(ARCH))
+GENTOO_CPU = arm
+GENTOO_ARCH = armv7a
+endif
+ifeq ($(findstring $(ARCH),armada370 armadaxp),$(ARCH))
+GENTOO_CPU = arm
+GENTOO_ARCH = armv7a_hardfp
+endif
+ifeq ($(findstring $(ARCH),bromolow cedarview evansport x86),$(ARCH))
+GENTOO_CPU = x86
+GENTOO_ARCH = i686
+endif
+ifeq ($(findstring $(ARCH),qoriq),$(ARCH))
+GENTOO_CPU = ppc
+GENTOO_ARCH = ppc
+endif
+ifeq ($(strip $(GENTOO_ARCH)),)
+$(error Arch $(ARCH) not supported)
+endif
+
+GENTOO_STAGE3 := $(shell curl -s http://distfiles.gentoo.org/releases/$(GENTOO_CPU)/autobuilds/latest-stage3-$(GENTOO_ARCH).txt | sed '/^\#/d' | awk '{print $$1}')
+GENTOO_STAGE3_URL := http://distfiles.gentoo.org/releases/$(GENTOO_CPU)/autobuilds/$(GENTOO_STAGE3)
+GENTOO_STAGE3_FILE := $(shell basename $(GENTOO_STAGE3))
+
+include ../../mk/spksrc.spk.mk
+
+.PHONY: gentoo-chroot_extra_install
+gentoo-chroot_extra_install: $(STAGING_DIR)/share/requirements.pybundle
```

Comment:
You might have overlooked this: `$(STAGING_DIR)/share/requirements.pybundle` can be removed.